docs/source/api/methods.rst

```diff
@@ -1,50 +1,60 @@
-****************
+*****************
 Analysis Methods
-****************
+*****************
+
+
+Core measurements
+-----------------
 
 Density
-*******
+^^^^^^
 
 .. autoapimodule:: density_calculator
     :members:
     :no-private-members:
     :no-special-members:
 
 Speed
-*****
+^^^^^^
 
 .. autoapimodule:: speed_calculator
     :members:
     :no-private-members:
     :no-special-members:
 
 Flow
-****
+^^^^
 
 .. autoapimodule:: flow_calculator
     :members:
     :no-private-members:
     :no-special-members:
 
-Profiles
-********
 
-.. autoapimodule:: profile_calculator
+
+Pedestrian Dynamics
```

Comment:
Why pedestrian dynamics? The other measures are also for pedestrian dynamics. In a past discussion we decided for: further measures as headline.